web/cobrands/sass/_dashboard.scss

```diff
@@ -297,8 +297,12 @@
 
 .dashboard-filters {
     .multi-select-button {
-        height: 28px;
-        line-height: 100%;
+        max-width: 10em;
+    }
+
+    .form-control {
+        // To make them have the same height as the .multi-select-button
+        height: 3.3em;
```

Comment:
This is fine - something else bit odd to think about later, as we now have select.form-control { height: 2.2em }, .filters .form-control { height: 2.2em }, .report-list-filters .form-control { height: 1.8em } and now with this .dashboard-filters .form-control { height: 3.3em }. And the filters on the heatmap page and the triage page do not look as nice as the public filters, I don't think they were updated to match the public around page.